src/main/java/org/reactome/release/qa/check/NewRegulationChecker.java

```diff
@@ -0,0 +1,84 @@
+
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.persistence.MySQLAdaptor;
+import org.reactome.release.qa.annotations.SliceQATest;
+import org.reactome.release.qa.common.AbstractQACheck;
+import org.reactome.release.qa.common.QACheckerHelper;
+import org.reactome.release.qa.common.QAReport;
+
+/**
+ *  This QA check finds any ReactionlikeEvents that have received a new, unreviewed Regulation instance (via 'regulatedBy' attribute).
+ * @author jcook
+ */
+
+import java.util.*;
+
+@SliceQATest
+public class NewRegulationChecker extends AbstractQACheck implements ChecksTwoDatabases
+{
+
+    private MySQLAdaptor priorAdaptor;
+
+    @Override
+    public String getDisplayName()
+    {
+        // Output report file name.
+        return "ReactionlikeEvent_Regulations_Not_Reviewed";
+    }
+
+    @Override
+    public void setOtherDBAdaptor(MySQLAdaptor adaptor)
+    {
+        this.priorAdaptor = adaptor;
+    }
+
+    @Override
+    public QAReport executeQACheck() throws Exception
+    {
+        QAReport report = new QAReport();
+        report.setColumnHeaders("DBID","DisplayName", "SchemaClass", "Issue", "MostRecentAuthor");
+        // Find all ReactionlikeEvents that have a filled 'regulatedBy' attribute in current slice.
+        Set<GKInstance> currentRlEsWithRegulations = getRlEsWithRegulationInstances(dba);
+
+        for (GKInstance currentRlE : currentRlEsWithRegulations) {
+            GKInstance previousRlE = priorAdaptor.fetchInstance(currentRlE.getDBID());
+            // QA check
+            if (newRegulationWithoutReviewed(currentRlE, previousRlE)){
+                report.addLine(currentRlE.getDBID().toString(), currentRlE.getDisplayName(), currentRlE.getSchemClass().getName(), "ReactionlikeEvent with new regulatedBy instance but has not yet been reviewed", QACheckerHelper.getLastModificationAuthor(currentRlE));
+            }
+        }
+        return report;
+    }
+
+    /**
+     * This method checks to see if a new regulation instance has been added to the ReactionlikeEvent by comparing between slices.
+     * If it does, it then compares the number of reviewed instances between slices. If the number of regulation instances is greater AND
+     * the number of reviewed instances is the same, it is flagged to indicate that this ReactionlikeEvent still needs to reviewed before
+     * it can be released.
+     * @param currentRlE GKInstance -- ReactionlikeEvent from current slice.
+     * @param previousRlE GKInstance -- Same ReactionlikeEvent, but from previous slice.
+     * @return boolean -- True if a new, unreviewed regulation instance has been added.
+     * @throws Exception -- DBA exceptions.
+     */
+    private boolean newRegulationWithoutReviewed(GKInstance currentRlE, GKInstance previousRlE) throws Exception {
+        // If previousRlE does not exist, we ignore it since that does not pertain to this particular QA check.
+        if (previousRlE != null) {
```

Comment:
You could consider a guard clause here to return if the `previousRlE` is null.  

```
if (previousRlE == null) {
    return false;
}
```

It would reduce the indentation of the main logic in the if block.  It's a style judgement call, though.

Reply:
I don't mind extra indentation, I like having fewer exit points

Reply:
I think you could keep the number of exit points the same if you were to use the guard clause and change line 74 to:

```
return currentRlERegulationCount > previousRlERegulationCount && 
currentRlEReviewedCount == previousRlEReviewedCount
```

In this case, though, the benefit is small, so I'm okay with leaving it as is.

Guard clauses are usually considered good practice, though.  This article addresses the argument of not using guard clauses due to too many exit points: https://medium.com/better-programming/refactoring-guard-clauses-2ceeaa1a9da.  The counter-argument they give is with guard clauses, all the additional exit points are at the beginning of the method and show the conditions under which the "normal" logic isn't followed.  It can also reduce indentation and conditionals within conditionals making code hard to reason about.  The second point isn't relevant in this case since it's only one level of nesting, but it shows up a lot of the time.

I try to use guard clauses when it helps reduce the code complexity and improve readability since it is a well established refactoring principle: https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html

Reply:
Yea that change makes sense, I have made it